lib/broccoli/bundler.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const concatFiles = require('./concat-files');
+const processModulesOnly = require('./babel-process-modules-only');
+
+class Bundler {
```

Comment:
`module.exports =` at the `class` keyword please, that way its easy to see (at the definition) if something is exported or not